frontend/src/components/notifications/NotificationsTab.vue

```diff
@@ -4,11 +4,11 @@
       <v-row>
         <v-col class="mt-3 ml-3">
           <div>
-            <AppButton @click="toggleMarkUnreadButtonInNotificationTable(false)" variant="text">
+            <AppButton class="mx-1 notifications-button" size="small" :primary="false" @click="toggleMarkUnreadButtonInNotificationTable(false)">
```

Comment:
it would be great if we can keep the distance between text and icon the same for all buttons (in Messages and Notifications)